src/MatrixTypes.f90

```diff
@@ -32,6 +32,7 @@
 !>  - @ref BLAS2 "BLAS2": @copybrief BLAS2
 !>  - @ref BLAS3 "BLAS3": @copybrief BLAS3
 !>
+!> TIBWSFB: This documentation doesnt match the current interface
```

Comment:
I like it but didn't you just change this entire interface... doesn't that mean you should have updated the documentation ;-)